lib/css/components/_stacks-popovers.less

```diff
@@ -0,0 +1,250 @@
+//
+//  STACK OVERFLOW
+//  POPOVERS
+//
+//  This CSS comes from Stacks, our CSS & Pattern library for rapidly building
+//  Stack Overflow. For documentation of all these classes and how to contribute,
+//  visit https://stackoverflow.design/
+//
+//  TABLE OF CONTENTS
+//  • VARIABLES
+//  • BASE STYLE
+//  • ARROWS
+//  • LOCATIONS
+//
+//  ============================================================================
+//  $   VARIABLES
+//  ----------------------------------------------------------------------------
+@popover-background: @white;
+
+
+//  ============================================================================
+//  $   BASE STYLES
+//  ----------------------------------------------------------------------------
+.s-popover {
+    visibility: hidden;
+    position: relative;
+    min-width: 12rem;
+    max-width: 24rem;
+    width: 100%;
+    padding: @su12;
+    z-index: @zi-hide;
+    opacity: 0;
+    border-radius: @br-md;
+    border: 1px solid @black-100;
+    background-color: @white;
+    box-shadow: @bs-sm;
+    color: @fc-dark;
+    font-size: @fs-body1;
+
+    will-change: visibility, z-index, opacity; // Not supported in IE11 / Edge
+    transition: opacity 100ms @te-smooth 0s, z-index 0s 100ms, visibility 0s 100ms; // Transition out
+
+    &[aria-hidden="false"] {
+        visibility: visible;
+        opacity: 1;
+        z-index: @zi-popovers;
+        transition: opacity 100ms @te-smooth 0s, z-index 0s 0s, visibility 0s 0s; // Transition in
+    }
+}
+
+//  ============================================================================
+//  $   CLOSE BUTTON
+//  ----------------------------------------------------------------------------
+
+.s-popover--close {
+    position: absolute !important;
+    top: @su4;
+    right: @su4;
+    padding: @su8 !important;
+}
+
+
+//  ============================================================================
+//  $   ARROWS
+//  ----------------------------------------------------------------------------
+//
+//  TABLE OF CONTENTS
+//  • BASE STYLE
+//  • LOCATIONS
+//      • TOP
+//      • BOTTOM
+//      • TOP & BOTTOM DIRECTIONS (Left, Center, Right)
+//      • LEFT
+//      • RIGHT
+//      • LEFT & RIGHT DIRECTIONS (Top, Center, Bottom)
+//
+//
+//  ============================================================================
+//  $   VARIABLES
+//  ----------------------------------------------------------------------------
+@arrow-width:                @su8;
```

Comment:
Optimally these variables should not be in the global scope. If that's unavoidable, they should be at least namespaced with something like `@stacks-internals-...`.